docs/install/airgapped-clusters.md

```diff
@@ -1,259 +1,299 @@
+
+##### Generate Manifests
+
+1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. In addition, define and export a `KUBERNETES_VERSION` environment variable, where the value will be the exact version of your Kubernetes cluster where Ondat is going to be deployed - for example, `v1.23.5`. Lastly, define and export a `ETCD_ENDPOINTS` environment variable, where the value will be a list of `etcd` endpoints as comma-separated values.
 
     ```bash
-    cd storageos-dry-run
-    vi storageos-operator.yaml
+    export STORAGEOS_USERNAME="storageos"
+    export STORAGEOS_PASSWORD="storageos"
+    export KUBERNETES_VERSION="v1.23.5"
+    export ETCD_ENDPOINTS="203.0.113.10:2379,203.0.113.11:2379,203.0.113.12:2379"
     ```
 
-    Edit the file `storageos-operator.yaml` and change:
-    - Find the `ConfigMap` called `storageos-related-images` and change the
-      URLs of the images adding your registry URL prefix.
-    - Find the `Deployment` called `storageos-operator` and change the `images`
-      of the 2 containers on it adding your registry URL prefix. They are the
-      containers `manager` and `kube-rbac-proxy`.
+1. Run the following  `kubectl-storageos` plugin command with the `--dry-run` flag to generate the Ondat deployment Kubernetes manifests in a directory, called `storageos-dry-run`.
 
-1. (Optional) Amend the file `storageos-cluster.yaml`
-    The StorageOSCluster definition depends on your cluster. For all available
-    options, check the [operator
-    reference](/docs/reference/cluster-operator/configuration). You can add
-    options such as tolerations, nodeSelectors, etc.
+    ```bash
+    kubectl storageos install \
+      --dry-run \
+      --skip-etcd-endpoints-validation \
+      --etcd-endpoints="$ETCD_ENDPOINTS" \
+      --k8s-version="$KUBERNETES_VERSION" \
+      --admin-username="$STORAGEOS_USERNAME" \
+      --admin-password="$STORAGEOS_PASSWORD"
+    ```
 
-## Step 4. Pulling images into a private registry
+1. To review the list of manifests generated in the newly created `storageos-dry-run` directory, run the following commands.
 
-The images set on the previous steps need to be added to your registry. You can
-find them with:
+ ```bash
+ cd storageos-dry-run/
+ ls
+ ```
+
+### Step 4 - Push Ondat Images To Private Registry
+
+ 1. Get the list of all the container images required for Ondat to be deployed successfully and push them to your private registry which will be accessible through your air-gapped environment.
 
 ```bash
-grep -E  "RELATED|image:" *.yaml
+grep --extended-regexp "RELATED|image:" *.yaml
 ```
 
-## Step 5. Install Ondat
+### Step 5 - Modify Deployment Manifests
 
-```bash
-# Create the operators and CRDs first
-find . -name '*-operator.yaml'  | xargs -I{} kubectl create -f {}
+ 1. **`etcd-operator`** - Modify the `2-etcd-operator.yaml` manifest and apply the following changes.
```

Comment:
is this supposed to be 1, 2, 3 (so it should be 1.1.1.?)

Reply:
fixed this and the indentation for the code snippets, thanks!